### DISCUSSION TITLE
How to discuss details on a certain branch

### DISCUSSION THREAD

**Discussion by rendetto** [nortikin/sverchok#4857] [Q&A] [answered]
I'm not finding a way to discuss things on a particular development branch of Sverchok. Is there a way to do that ..is it even appropriate?
For example I have some changes/fixes proposal but the only way I'm finding is to create a PR.

Thanks

**Comment by portnov** [comment, marked as answer]
You mean you want to discuss someone else's changes, that are pushed to some branch, but for which there is no  PR yet?
Well, Github does not seem to suggest a specific way to do that. But you always can add comment to changes in any commit. Or you may ask the author of changes to create a PR, to discuss them there.

If there is already a PR for that branch in question, the appropriate place to discuss changes is that PR.

For less formal discussion, there is always Discord (mostly english speaking, but russian as well) and telegram chat (mostly russian speaking, but english as well).

Or I do not understand what are you talking about :)

**Reply by rendetto** [reply]
> But you always can add comment to changes in any commit. 

Oh... well ...found it at the bottom x_x
And yes @portnov , that was exactly what I was asking for :)
Thanks!

**Comment by portnov** [comment]
Btw, there is a feature in github, which is not totally obvious: you can create PR, and specify another (previously existing) PR as target, instead of master. We usually do not practice this, because our PRs usually do not tend to be longstanding. But we did this a couple of years ago, when we did a porting of Sverchok from Blender 2.79 to 2.80 in one mega-PR.

**Reply by rendetto** [reply]
Yes, I was thinking of creating a PR first but realized that a branch is a work in progress.... I had to scroll some more though :D
